README.md

```diff
@@ -239,6 +239,30 @@ The device plugin will initially discover all PCI network resources in the host
 4. "pfNames" - The Physical funtion name
```

Comment:
nit: function net device name

not part of this commit :) but maybe a followup for this  ?

Reply:
I will consider changing it in a followup commit